src/test/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResourceIntegrationTest.java

```diff
@@ -81,7 +81,7 @@ public void unknownCommand() {
           .body(
               "errors[0].message",
               startsWith(
-                  "Provided command unknown: \"unknownCommand\" not one of \"CollectionCommand\"s: known commands are [countDocuments,"));
+                  "Provided command unknown: \"unknownCommand\" not one of \"CollectionCommand\"s: known commands are [addIndex, countDocuments,"));
```

Comment:
When an unknown command name is passed to an endpoint, it returns list of supported commands. The commands are alphabetically added. Since we added new command `addIndex`, it became the first in the list of supported commands returned. Fixed the test case that checks for this message.

Reply:
I see.